addons/pos_self_order/static/src/app/services/self_order_service.js

```diff
@@ -202,6 +209,19 @@ export class SelfOrder extends Reactive {
         this.currentOrder.removeOrderline(line);
     }
 
+    async syncPresetSlotAvaibility(preset) {
+        try {
+            const presetAvailabilities = await rpc(`/pos-self-order/get-slots`, {
+                access_token: this.access_token,
+                preset_id: this.currentOrder?.preset_id?.id,
+            });
+
+            preset.computeAvailabilities(presetAvailabilities);
+        } catch {
+            console.info("Offline mode, cannot update the slot avaibility");
```

Comment:
What if the error is not a ConnectionLostError ?

Reply:
I added a local computation